perf/benchmarkapps/GrpcClient/Program.cs

```diff
@@ -312,12 +312,12 @@ private static void CalculateStatistics()
         Log($"Total errors: {errors}");
 
         BenchmarksEventSource.Register("grpc/firstrequest", Operations.Max, Operations.Max, "First Request (ms)", "Time to first request in ms", "n2");
-        BenchmarksEventSource.Register("grpc/rps/max;http/rps/mean", Operations.Max, Operations.Sum, "Max RPS", "RPS: max", "n0");
+        BenchmarksEventSource.Register("grpc/rps/mean;http/rps/mean", Operations.Max, Operations.Sum, "Requests/sec", "Requests per second", "n0");
```

Comment:
Just checking: Operations.Max is still the most appropriate here?

Reply:
It's consistent with what is in h2load.